dashboard/src/modules/components/AuthComponent/common-components.jsx

```diff
@@ -117,9 +117,12 @@ export const AuthForm = () => {
       <CardFooter>
         <div className="log-in-alternate">Or log in with...</div>
         <div className="alternate-btn-wrapper">
-          <Button variant="secondary">Red Hat SSO</Button>
-          <Button variant="secondary">GitHub</Button>
-          <Button variant="secondary">Gmail</Button>
+          <Button
+            variant="primary"
+            onClick={() => {dispatch(authenticationRequest())}}
+          >
+            Pbench OpenId
```

Comment:
I wonder if we should have a "name" configuration setting along with the URI and such, which the client could display here? If we're not going to do that, we should probably drop the "Pbench" and just label it an "OpenID Connect" server/broker. (Not for this PR, but something to consider.)

Reply:
I haven't tried to load this into my browser, but I think what we want is a button which is labeled basically "click here to log in".  At this point in the flow, there is only one login method (I believe...) which is the OIDC broker -- we don't have to name it anything...it's The Way in.  So, yeah, we can drop the "Pbench" if we want, and we can drop the "OpenId", because, on this page, that's an implementation detail which we don't need to mention.  Clicking the button will bring the User to some sort of OIDC page (or perhaps something altogether different -- the Pbench Server configuration will dictate where it goes, but it won't have any control over what gets loaded from there!), and then it will be out of our hands until the redirect brings the browser back.